packages/dds/map/src/mapKernel.ts

```diff
@@ -730,29 +754,42 @@ export class MapKernel implements IValueTypeCreator {
                 submit: (op: IMapValueTypeOperation, localOpMetadata: unknown) => {
                     this.submitMessage(op, localOpMetadata);
                 },
+                rebaseOp: (op: IMapValueTypeOperation, localOpMetadata: unknown) => {
+                    assert(false, "not implemented");
```

Comment:
Maybe put a bit more unique message such that it easy to find it in the code when we hit it.